velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.cpp

```diff
@@ -16,6 +16,7 @@
 #include "HdfsFileSystem.h"
 #include <hdfs/hdfs.h>
 #include "HdfsReadFile.h"
+#include "HdfsWriteFile.h"
```

Comment:
Use full path for includes, e.g. "velox/connectors/hive/storage_adapters/hdfs/HdfsWriteFile.h"